storage/libradb/src/event_store/test.rs

```diff
@@ -160,24 +160,19 @@ fn traverse_events_by_access_path(
 }
 
 fn arb_event_batches() -> impl Strategy<Value = (Vec<AccessPath>, Vec<Vec<ContractEvent>>)> {
-    (
-        vec(any::<AccountAddress>(), 3),
-        hash_set(any::<Vec<u8>>(), 3),
-        (0..100usize),
-    )
-        .prop_flat_map(|(addresses, event_paths, num_batches)| {
-            let all_possible_access_paths = addresses
+    // TODO: Get rid of the unnecessary prop_flat_map here.
+    (vec(any::<AccountAddress>(), 3), (0..100usize))
+        .prop_flat_map(|(addresses, num_batches)| {
+            let event_key_strategy = Union::new(addresses.clone().into_iter().map(Just));
+
+            let access_path_vec = addresses
```

Comment:
and this can be `event_key_vec`

and `all_possible_access_paths` seen later can be `all_possible_event_keys`